PurchasesCoreSwift/Misc/ISOPeriodFormatter.swift

```diff
@@ -11,7 +11,7 @@ import StoreKit
 
 @available(iOS 11.2, macOS 10.13.2, tvOS 11.2, *)
 
-class ISOPeriodFormatter {
+struct ISOPeriodFormatter {
```

Comment:
In this case, where there are no properties, I think we should default to class. Structs are great for things that store data, but when we're dealing with things that generally act on data, I think a class is preferred. When we have classes we're using pass-by-reference and this is generally favorable for objects that act on data. When you're building objects/models that store data, struct is generally your go-to solution, unless the struct needs to mutate its internal state, then it gets more nuanced.